portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,14 +26,33 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String[] wordArray = {"Hey!", "Check", "here!"};
+    /**
     String toJson = convertToJson(wordArray);
-
     response.setContentType("application/json;");
     response.getWriter().println(toJson);
+    */
   }
-  private String convertToJson(String[] wordArray) {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String getMessage = getParameter(request, "comment", "");
```

Comment:
For string constants (and other constants like integers as well), it is usually good to use a constant class member variable, e.g.:
  (public|private) final String COMMENT = "comment";
 
so that on the server side, whenever you need to reference the constant string "comment", you can use the "COMMENT" variable. Roughly speaking, think constants as those whose content does not change over the whole lifecycle of the server. Note that in Java, constant names are conventionally all capitalized.